appengine-jetty-managed-runtime/src/main/docker/resources/jetty-logging.properties

```diff
@@ -0,0 +1,2 @@
+# Direct Jetty logging to JavaUtilLog
+org.eclipse.jetty.util.log.class=org.eclipse.jetty.util.log.JavaUtilLog
```

Comment:
I added this resource to ensure that jetty logging is directed to java util logging